src/triage/component/postmodeling/contrast/evaluator.py

```diff
@@ -0,0 +1,543 @@
+"""
+Model Evaluator
+
+This script contain a set of elements to help the postmodeling evaluation
+of audited models by triage.Audition. This will be a continuing list of
+routines that can be scaled up and grow according to the needs of the
+project, or other postmodeling approaches.
+
+To run most of this routines you will need:
+    - S3 credentials (if used) or specify the path to both feature and
+    prediction matrices.
+    - Working database db_engine.
+"""
+
+import pandas as pd
+import numpy as np
+from sqlalchemy.sql import text
+from matplotlib import pyplot as plt
+from sklearn import metrics
+from collections import namedtuple
+
+from utils.file_helpers import download_s3
+from utils.test_conn import db_engine
+from utils.aux_funcs import recombine_categorical
+
+# Get individual model_ids from Audition outcome
+
+
+def get_models_ids(audited_model_group_ids):
+    query = db_engine.execute(text("""
+    SELECT model_group_id,
+           model_id
+    FROM model_metadata.models
+    WHERE model_group_id = ANY(:ids);
+    """), ids=audited_model_group_ids)
+
+    ModelTuple = namedtuple('Model', ['model_group_id', 'model_id'])
+    l = [ModelTuple(*i) for i in query]
+
+    return(l)
```

Comment:
It would be best not to (re)-define this class, `ModelTuple`, upon every invocation of the utility; (this way, it's not pickleable, because neither `pickle` nor anyone else can access the model definition, *etc.*).

I dunno if there could be a more descriptive name for the class, as well -- perhaps `ModelMeta`?

You should also ensure that the `__name__` field you specify, `Model`, matches the reference you create in scope, `ModelTuple`.

(Also, the `_make` method of `namedtuple` is useful.)

*E.g.*:

    ModelMeta = namedtuple('ModelMeta', ('model_group_id', 'model_id'))

    def get_model_ids(...):
        return [
            ModelMeta._make(row) for row in db_engine.execute(
                ...
            )
        ]